woocommerce/woocommerce-ios
# Move URL+Woo to WooConstants

Refs #2245 

As pointed to by @pmusolino in https://github.com/woocommerce/woocommerce-ios/pull/2245#discussion_r422684024, I didn't know that URLs in `WooConstants` existed. 😅 

This moves the URL+Woo file contents to `WooConstants`. I also changed the forcefully unwrapped declarations to use `trustedURL()` so (hopefully) we'll get better crash errors. Highly unlikely for this but I think this is a better practice than `!`.  

## Testing

Please do a quick regression test of one of the URLs. For example:

1. Navigate to Settings
2. Tap on Feature Request. 
3. Confirm that a browser is opened with the correct feature request URL. 

## Reviewing 

Only 1 reviewer is needed but anyone can review. 

## Submitter Checklist
